.github/workflows/build-and-publish.yml

```diff
@@ -26,6 +26,7 @@ jobs:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
           latest: 1   # this should only be set on the current minor branch
+          build_ml: all  # build ml,gnn and gnn-gpu
```

Comment:
change looks good. Tested on my master branch and the action recognized the input correctly.